qcodes/tests/dataset/test_measurement_context_manager.py

```diff
@@ -217,6 +218,34 @@ def get_raw(self):
     SA.close()
 
 
+@pytest.fixture
+def meas_with_registered_param(DAC, DMM):
+    meas = Measurement()
+    meas.register_parameter(DAC.ch1)
+    meas.register_parameter(DMM.v1, setpoints=[DAC.ch1])
+    yield meas
+
+@pytest.mark.usefixtures("experiment")
```

Comment:
just decorate the meas_with_registered_param fixture itself and you won't need to repeat it for all the tests :)